recipes-bsp/barebox/barebox.inc

```diff
@@ -82,6 +82,10 @@ do_install() {
 
 # Suffix that allows to create different barebox images in one BSP
 BAREBOX_IMAGE_SUFFIX = ""
+# This allows a machine.conf to specify the required images to install instead
+# of installing all found images
+BAREBOX_IMAGES ??= "*.img"
+BAREBOX_IMAGES_TO_DEPLOY = "${@' '.join('${B}/images/%s' % i for i in d.getVar('BAREBOX_IMAGES').split())}"
```

Comment:
I think that will fail if you have spaces in your file name. Not sure if we want to support that anyway.
Maybe rename `BAREBOX_IMAGES` to `BAREBOX_MULTI_IMAGES`? Might make it clearer that this only applied to multi-image (non-multi-image, e.g. `barebox.efi` is installed top-level).

Reply:
I didn't see images containing spaces..

Rename it to `BAREBOX_MULTI_IMAGES` sounds reasonable but isn't it to specific? Isn't it the case that all images are stored within images except for `barebox.efi`. `BAREBOX_MULTI_IMAGES` assumes that it is only for multi-images but you can specify it for normal images too even if it makes no sense.

Reply:
I thought non multi-image barebox binaries are installed top level in the build directory as well. Just tried it out and they are under images/ as well, so no objection from my side.

Re: spaces: Just raising awareness. Change is fine with me.